FEATURE-LIFECYCLE.md

```diff
@@ -10,42 +10,42 @@ information at [Feature Lifecycle Checklist](FEATURE-LIFECYCLE-CHECKLIST.md).
 <table>
   <tr>
     <td></td>
-    <td>Development (pre-alpha)</td>
+    <td>Experimental</td>
     <td>Alpha</td>
     <td>Beta</td>
     <td>Stable</td>
   </tr>
   <tr>
     <td>Purpose</td>
-    <td>Active development of a feature, not for use by end users directly.</td>
+    <td> Feature is under active development and user facing APIs will most likely change. Users should deploy experimental features with extreme caution, preferably in non-production environments as newer versions can require a migration effort.</td>
     <td>Used to get feedback on a design or feature or see how a tentative design performs, etc. Targeted at developers and expert users.</td>
     <td>Used to vet a solution in production without committing to it in the long term, to assess its viability, performance, usability, etc. Targeted at all users.</td>
     <td>Dependable, production hardened</td>
   </tr>
   <tr>
     <td>Stability</td>
-    <td>Unknown</td>
+    <td>May be buggy. Using the feature may expose bugs. Users must opt-in to use feature.</td>
```

Comment:
If Experimental and Alpha have the same stability, then I'd just copy line 28.

Another approach is to take a riff off what is there, which is `May be buggy. This feature may have limited testing and field experience. Not active by default.`

This other approach represents a development feature as really green, (not field proven..)